### PR TITLE
Unquarantine but skip on OSX all template tests

### PR DESCRIPTION
**Pull request by HaoK**
We think the template tests are ok everywhere but OSX where there's a cert issue going on, so switch to targeted isolation instead of social distancing

### CODE REVIEW

**Comment by Pilchie** [comment]
:eyes:

File: .azure/pipelines/ci.yml [hunk 1]
```diff
@@ -691,7 +691,7 @@ stages:
 
   - template: jobs/default-build.yml
     parameters:
-      condition: notin(variables['Build.Reason'], 'PullRequest')
+      #condition: notin(variables['Build.Reason'], 'PullRequest')
```
Comment (HaoK) [inline .azure/pipelines/ci.yml:694]: Revert this before merge

**Comment by Pilchie** [comment]
It looks like there have been cert issues on Windows as well as on Mac, so I'm not sure this is the right solution.

**Comment by HaoK** [comment]
I guess the other thing we could consider is limit these tests only to known good queues, WIndows10 and Ubuntu?

**Pull request closed without merging**